website_blog_layout_options/README.rst

```diff
@@ -0,0 +1,65 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+===========================
+website_blog_layout_options
+===========================
+
+
+* Adds 2 extra display types of posts to the post list of any blog:
+
+1. Teaser: The title of the post with a small teaser and a "read more" link.
```

Comment:
btw. if you want all of this being in the scope of the bullet point above, you need indents